.bbtravis.yml

```diff
@@ -72,6 +72,8 @@ matrix:
 # Dependencies installation commands
 install:
   - pip install -U pip
+  # pycparser wheel is broken, and sometime mistakingly re-publish the wheels
+  - pip install --no-binary pycparser
```

Comment:
This looks like it has been fixed upstream: https://github.com/eliben/pycparser/issues/151 ?